docs/src/main/asciidoc/amazon-lambda-http.adoc

```diff
@@ -3,14 +3,14 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= Amazon Lambda with RESTEasy, Undertow, or Vert.x Web
+= Amazon Lambda with RESTEasy, Undertow, or Reactive Routes
```

Comment:
Ok, so this should be the name of the quarkus extension, right? If so then +1 and thanks for spotting this. I wasn't sure if it refers to the "real" Vert.x Web, i.e. `io.vertx:vertx-web`. Apparently, the original name of the extension got really confusing after the whole http engine was built on top of Vert.x.

Reply:
I know the original `io.vertx:vertx-web` is refered in some guide but for the function support it's reactive routes. I too was confused by the usage of vertx-web ;)

Reply:
Yes, the point is that when this extension was created it was an integration of `io.vertx:vertx-web`...